sway-core/src/language/call_path.rs

```diff
@@ -100,7 +100,7 @@ impl CallPath {
     ///
     /// Paths to _external_ libraries such `std::lib1::lib2::my_obj` are considered full already
     /// and are left unchanged since `std` is a root of the package `std`.
-    pub fn to_fullpath(&self, namespace: &mut Namespace) -> CallPath {
+    pub fn to_fullpath(&self, namespace: &Module) -> CallPath {
```

Comment:
Why is this no longer mutable?
Not saying it's wrong but I'd like to understand the reasoning for the change.

Reply:
It didn't need to be mutable in the first place :)